conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -159,7 +160,9 @@ def requests_ca_bundle_check(prefix: str, verbose: bool) -> None:
     requests_ca_bundle = os.getenv("REQUESTS_CA_BUNDLE")
     if not requests_ca_bundle:
         return
-    elif not Path(requests_ca_bundle).exists():
+    elif Path(requests_ca_bundle).exists():
+        print(f"{OK_MARK} `REQUESTS_CA_BUNDLE` was verified.\n")
+    elif re.match("^https?://", requests_ca_bundle) is None:
```

Comment:
This results in the `$REQUESTS_CA_BUNDLE` not being validated (see the `else` clause). We need something like the following instead:

```suggestion
    elif not (
        Path(requests_ca_bundle).exists()
        or re.match("^https?://", requests_ca_bundle)
    ):
```

We also have a `conda.common.url.is_url` convenience function that can be used:

```suggestion
    elif not (
        Path(requests_ca_bundle).exists()
        or is_url(requests_ca_bundle)
    ):
```

Reply:
Thanks for showing me `is_url`. I've added it in the latest commit. I've also changed the order of the `if` to hopefully make things more straightforward:

1. Is it a URL ? Yes → defer to `requests`
2. Otherwise, does the file exist ? Yes → we're good
3. Otherwise (i.e. the file doesn't exist); show an error